fastocr/service.py

```diff
@@ -53,9 +56,62 @@ async def basic_general(self, image: bytes, lang=''):
     async def close(self):
         await self.session.close()
 
+class YoudaoOcr(BaseOcr):
+    API_BASE = 'https://openapi.youdao.com/ocrapi'
+    CURTIME = str(int(time))
```

Comment:
Maybe `str(time())` ?

Reply:
Youdao use `int()` method in their demos, the test results seems to be OK and I have no idea that if `int()` is necessary. Anyway, I will remove it.

Reply:
Youdao use `int()` method in their demos, the test results seems to be OK and I have no idea that if `int()` is necessary. Anyway, I will remove it.